src/main/java/me/jellysquid/mods/sodium/client/world/WorldSlice.java

```diff
@@ -389,15 +389,15 @@ public static int getLocalChunkIndex(int x, int z) {
     // Modern checks if the sky is darkened, which only happens in the nether. However, I think 1.7.10's hasNoSky is
     // close enough and possibly a behavior change between versions. I also don't know why it's rotationally asymmetric
     public float getBrightness(ForgeDirection direction, boolean shaded) {
+
         if (!shaded) {
             return world.provider.hasNoSky ? 0.9f : 1.0f;
         }
         return switch (direction) {
-            case DOWN -> world.provider.hasNoSky ? 0.9f : 0.5f;
-            case UP -> world.provider.hasNoSky ? 0.9f : 1.0f;
+            case DOWN -> 0.5f;
```

Comment:
Why this change?

Reply:
Matches Vintagium's rendering. Dunno if it's required, but it works and it's a but cleaner.